tests/conv/layer_example.c

```diff
@@ -147,7 +158,7 @@ int main(int argc, char* argv[])
   if (argc > i) fuse_type = atoi(argv[i++]);
   if (argc > i) bc = atoi(argv[i++]);
   if (argc > i) bk = atoi(argv[i++]);
-  if (argc > i) prec_bf16 = atoi(argv[i++]);
+  if (argc > i) prec = atoi(argv[i++]);
```

Comment:
better to use char and then mapping the libxsmm_datatype as this solution doesn't scale to HF8

Reply:
strcmp is messy. postponed this change to a later date